recipe/0003-pycryptosat-remove-quote-from-macro.patch

```diff
@@ -0,0 +1,31 @@
+--- python/setup.py.in
++++ python/setup.py.in
+@@ -125,7 +125,7 @@
+ modules = dict(
+     name = "pycryptosat",
+     sources = ["${CMAKE_CURRENT_SOURCE_DIR}/src/pycryptosat.cpp"],
+-    define_macros = [('LIBRARY_VERSION', '"' + __LIBRARY_VERSION__ + '"')],
++    define_macros = [('LIBRARY_VERSION', __LIBRARY_VERSION__)],
+     extra_compile_args = cconf + ['-I${PROJECT_SOURCE_DIR}', '-I${PROJECT_BINARY_DIR}/cmsat5-src'],
+     extra_link_args = extra_link_args,
+     language = "c++",
+--- python/src/pycryptosat.cpp
++++ python/src/pycryptosat.cpp
+@@ -23,6 +23,8 @@
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+ **********************************/
++#define XSTR(x) #x
++#define STR(x) XSTR(x)
+ 
+ #include <Python.h>
+ #include <structmember.h>
+@@ -1103,7 +1105,7 @@
+ 
+     Py_INCREF(&pycryptosat_SolverType);
+     PyModule_AddObject(m, "Solver", (PyObject *)&pycryptosat_SolverType);
+-    PyModule_AddObject(m, "__version__", PyUnicode_FromString(LIBRARY_VERSION));
++    PyModule_AddObject(m, "__version__", PyUnicode_FromString(STR(LIBRARY_VERSION)));
```

Comment:
@msoos: Instead using this crappy `STR(x)` workaround, it probably makes more sense to not use `define_macros` for this at all but just turn `pycryptosat.cpp` into a template `pycryptosat.cpp.in` and simply use
```cpp
    PyModule_AddObject(m, "__version__", PyUnicode_FromString("@CMS_FULL_VERSION@"));
```
or something like that here.